### DISCUSSION TITLE
What's the correct way of uninstalling?

### DISCUSSION THREAD

**Discussion by wh75er** [dcm4che/dcm4che#926] [Q&A] [answered]
I've never worked with maven and I didn't find anything on how to uninstall utilities from system after `mvn install` neither on google or maven's documentation. Can someone help me with that, please? Manual uninstallation by deleting binaries from /usr/bin seems stupid

**Comment by gunterze** [comment, marked as answer]
`mvn install` builds the binary artifacts and copy them in your local maven repository under `$USER/.m2/repository`, but does not install anything on your system.
